ads/google/a4a/performance.js

```diff
@@ -194,36 +224,63 @@ export class GoogleAdLifecycleReporter extends BaseLifecycleReporter {
    * @override
    */
   sendPing(name) {
-    this.emitPing_(this.buildPingAddress_(name));
+    this.emitPing_(this.buildPingAddress_(name, this.extraVariables_));
```

Comment:
`emitPing_` exists mostly for testing.  It allows me to spy on it in the tests and catch the fully constructed ping URL.